### PR TITLE
RA-250: Save playlist when it change

### PR DESCRIPTION
**Pull request by aameijide**

### CODE REVIEW

**Comment by aameijide** [comment]
DB View was already created.

File: VAS.Tests/Services/TestPlaylistController.cs [hunk 1]
```diff
@@ -38,13 +39,21 @@ public class TestPlaylistController
...
 		public void FixtureSetup ()
 		{
 			mockPlayerController = new Mock<IPlayerViewModel> ();
 			mockGuiToolkit = new Mock<IGUIToolkit> ();
+
+			storageManagerMock = new Mock<IStorageManager> ();
+			storageManagerMock.SetupAllProperties ();
+			storageMock = new Mock<IStorage> ();
+			storageManagerMock.Object.ActiveDB = storageMock.Object;
+			App.Current.DatabaseManager = storageManagerMock.Object;
```
Comment (ylatuya) [inline VAS.Tests/Services/TestPlaylistController.cs]: Why is this change needed at all ?

Comment (aameijide) [inline VAS.Tests/Services/TestPlaylistController.cs]: Playlist are saved when are created or modified, so we need to mock Database in test. If not, TestAddSamePlaylist and TestNewPlaylist fails
